exist-core/src/test/xquery/maps/mapsLookup.xql

```diff
@@ -13,7 +13,7 @@ function mlt:postfix_lookup_on_map() {
 };
 
 declare
-    %test:assertEquals("Jenna", "Scott")
+    %test:assertEqualsPermutation("Jenna", "Scott")
```

Comment:
That is a handy test annotation!

Reply:
Right - I had to look into the source code to find it - because the docs on exist-db.org are out of date :-(

Reply:
Is it documented or should we open a ticket on the documentation?

Reply:
It's documented. But eXist-db.org hasn't been updated :-(